lib/cadet_web/admin_controllers/admin_assets_controller.ex

```diff
@@ -22,7 +22,7 @@ defmodule CadetWeb.AdminAssetsController do
 
     case Assets.delete_object(Courses.assets_prefix(course_reg.course), foldername, filename) do
       {:error, {status, message}} -> conn |> put_status(status) |> text(message)
-      _ -> conn |> put_status(204) |> text('')
+      _ -> conn |> put_status(204) |> text(~c"")
```

Comment:
Hi, what is the reason for this change? This should be unrelated to this PR?

Reply:
Same as the other one it was auto-corrected while formatting